### ISSUE TITLE
add ToQuotes() for Heikin-Ashi

### ISSUE THREAD

**Issue by moslem7026** [DaveSkender/Stock.Indicators#872]
i wanted to change candles to HeikinAshi  and use `GetHeikinAshi` then use `ConvertToQuotes=`  to have list<quote> instead of list<HeikinAshiResult> . 

ConvertToQuotes has been replaced with ToQuotes() but  `lstQuote.GetHeikinAshi().ToQuotes();` return 

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS1061	'IEnumerable<HeikinAshiResult>' does not contain a definition for 'ToQuotes' and no accessible extension method 'ToQuotes' accepting a first argument of type 'IEnumerable<HeikinAshiResult>' could be found (are you missing a using directive or an assembly reference?)
```

**Comment by DaveSkender** [comment]
The `HeikenAshiResult` is based on `IQuote`.  What that means is it can be used as is as a direct replacement for quotes, no conversion needed.

With that said, it wouldn’t be difficult to extend `.ToQuotes()` to work on HeikinAshi if you just wanted to convert the class for cosmetic reasons.  I’m adding it to the backlog.

**Comment by DaveSkender** [comment]
This was implemented in v2.2.0
